### PR TITLE
Fixed the issue where core thread metrics were in effect(#11704)

### PR DESCRIPTION
**Pull request by robin977**
Fixed the issue where core thread metrics were in effect(#11704)
1，Core thread does not effect, by modifying the DefaultApplicationDeployer started method, after all the service start execution
2，Specify the name of the thread, modify the name in the DefaultExecutorRepository
3，Retain the FrameworkExecutorRepository sharedScheduledExecutor

### CODE REVIEW

File: dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java [hunk 1]
```diff
@@ -109,6 +110,7 @@ public synchronized ExecutorService createExecutorIfAbsent(URL url) {
             executors.put(executorCacheKey, executor);
         }
 
+        executorCacheKey= (CONSUMER_SIDE.equalsIgnoreCase(url.getParameter(SIDE_KEY)) ? "DubboClientHandler" : SERVER_THREAD_POOL_NAME ) + "-" + executorCacheKey;
```
Comment (songxiaosheng) [inline dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java:113]: please remove thread prefix  code  in  this  class

**Review by songxiaosheng — CHANGES_REQUESTED** [review]

File: dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java [hunk 2]
```diff
@@ -57,6 +57,7 @@
 import static org.apache.dubbo.common.constants.LoggerCodeConstants.COMMON_ERROR_USE_THREAD_POOL;
 import static org.apache.dubbo.common.constants.LoggerCodeConstants.COMMON_EXECUTORS_NO_FOUND;
 import static org.apache.dubbo.common.constants.LoggerCodeConstants.COMMON_UNEXPECTED_EXECUTORS_SHUTDOWN;
+import static org.apache.dubbo.config.Constants.SERVER_THREAD_POOL_NAME;
```
Comment (songxiaosheng) [inline dubbo-common/src/main/java/org/apache/dubbo/common/threadpool/manager/DefaultExecutorRepository.java:60]: image

**Review by songxiaosheng — CHANGES_REQUESTED** [review]
please fix unused import

**Review by songxiaosheng — APPROVED** [review]
LGTM

**Review by songxiaosheng — APPROVED** [review]

**Comment by songxiaosheng** [comment]
@AlbumenJ PTAL

**Review by AlbumenJ — APPROVED** [review]

**Pull request merged**